external/wiki/Proxying-Cockpit-over-NGINX.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Proxying Cockpit over NGINX
+source: https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-NGINX
+---
+
+404: Not Found
```

Comment:
@garrett : Whoops, this looks like a bug from your recent PR #526?

Reply:
Hahah, whoops.

Reply:
This actually seems to be a bug in Git Wiki.

Compare these pages...

Apache:

- Pretty: https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt
- Raw: https://raw.githubusercontent.com/wiki/cockpit-project/cockpit/Proxying-Cockpit-over-Apache-with-LetsEncrypt.md

NGINX:

- Pretty: https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-NGINX
- Raw: https://raw.githubusercontent.com/wiki/cockpit-project/cockpit/Proxying-Cockpit-over-NGINX.md

These are the URLs the script constructs and they should work. In the case of the pretty (rendered, GitHub UI) URLs, they both work. But the raw only works for Apache, not NGINX. Something seems broken on GitHub's side.

We could modify the page in some way and hope that it triggers GitHub to re-export the raw version of the file, then run the script. But that's all I can think of, as this URL API should work for grabbing the raw files.